dotcom-rendering/jest.config.cjs

```diff
@@ -1,12 +1,25 @@
 const swcConfig = require('./scripts/webpack/.swcrc.json');
 
+/** @type {import('jest').Config} */
 module.exports = {
 	testEnvironment: 'jsdom',
 	moduleDirectories: ['node_modules', 'src'],
+	moduleFileExtensions: [
+		'js',
+		'mjs',
+		'cts',
+		'cjs',
+		'jsx',
+		'ts',
+		'tsx',
+		'json',
+		'node',
+	],
```

Comment:
In other places I think you've referenced a `.ctsx` extension? Is that relevant here?

Reply:
I think I only referenced it as a potential match for [`/\.(c|m)?(t|j)sx?/i`](https://regex101.com/r/P6JHq1/1)? I was just trying to cover all bases, even if the file do not actually appear in the codebase…